docs/changes.md

```diff
@@ -0,0 +1,12 @@
+# News and Updates
+
+Here are some recent changes to our autograder platform. If you have any questions or issues with any of these changes, please email [help@gradescope.com](mailto:help@gradescope.com).
+
+## December 29th, 2020
+
+- The Ubuntu base image was upgraded from Ubuntu 18.04 LTS fo Ubuntu 20.04 LTS.
+
```

Comment:
I think after considering the possible ramifications it'll be safer to do this in the summer, but I do think that adding this page will be helpful. So let's just remove this section for now.

```suggestion
```